index.html

```diff
@@ -227,7 +228,7 @@ <h2>Amit</h2>
 			</div>
 
 			<div class="wow fadeInUp col-md-3 col-sm-6" data-wow-delay="2s">
-				<div class="trainer-thumb">
+				<div class=".trainer-thumb">

```

Comment:
The class attribute value should not start with a dot. It should be "<div className="trainer-thumb"></div>".